src/pages/profile/[id].tsx

```diff
@@ -0,0 +1,15 @@
+import { type NextPage } from "next";
+import {CreateBlogPostWidget} from "~/components/create-blog-post-widget";
+import { Layout } from "~/components/Layout";
+import {ProfileTabbedView} from "~/components/ProfileTabbedView";
+
+const Profile: NextPage = () => {
+  return (
+    <Layout>
+        <ProfileTabbedView />
+        <CreateBlogPostWidget />
```

Comment:
just checking: Is it intentional to put the createBlogPost button in this page?

Reply:
If so, creating a new blog post doesn't effect the '# of blog posts' in the profile page (the count query is not invalidated)

Reply:
this was intentional yes, but I'll work on updating the '# of blog posts' bug